### ISSUE TITLE
Failing to initialise vulkan

### ISSUE THREAD

**Issue by Schweeeeeeeeeeeeeeee** [ppy/osu#27904]
### Type

Crash to desktop

### Bug description

Vulkan renderer cant be enabled when restarting it says it failed to initialize

Maybe its my hardware?
![image](https://github.com/ppy/osu/assets/88291316/fcca3598-c6c7-4d6f-85a7-a978904967f3)

### Screenshots or videos

```
dotnet run --project osu.Desktop 
[runtime] 2024-04-16 19:35:14 [verbose]: Starting legacy IPC provider...
[runtime] 2024-04-16 19:35:14 [verbose]: 🖼 Configuration renderer choice: Deferred_Vulkan
[runtime] 2024-04-16 19:35:14 [verbose]: 🖼 Renderer fallback order: [ Vulkan (Experimental), OpenGL, OpenGL (Experimental) ]
[runtime] 2024-04-16 19:35:14 [verbose]: 🖼 Initialising "Deferred" renderer with "Vulkan" surface
[runtime] 2024-04-16 19:35:14 [verbose]: Updated display mode to desktop resolution: 2560x1440@165, SDL_PIXELFORMAT_RGB888
[runtime] 2024-04-16 19:35:14 [verbose]: 🖼 Renderer initialisation failed with:
[runtime] 2024-04-16 19:35:14 [verbose]: System.TypeInitializationException: The type initializer for 'Vulkan.VulkanNative' threw an exception.
[runtime] 2024-04-16 19:35:14 [verbose]: ---> System.DllNotFoundException: Unable to load shared library 'libdl' or one of its dependencies. In order to help diagnose loading problems, consider using a tool like strace. If you're using glibc, consider setting the LD_DEBUG environment variable:
[runtime] 2024-04-16 19:35:14 [verbose]: /home/boobs/osu/osu.Desktop/bin/Debug/net8.0/runtimes/linux-x64/native/libdl.so: cannot open shared object file: No such file or directory
[runtime] 2024-04-16 19:35:14 [verbose]: /usr/share/dotnet/shared/Microsoft.NETCore.App/8.0.4/libdl.so: cannot open shared object file: No such file or directory
[runtime] 2024-04-16 19:35:14 [verbose]: /home/boobs/osu/osu.Desktop/bin/Debug/net8.0/libdl.so: cannot open shared object file: No such file or directory
[runtime] 2024-04-16 19:35:14 [verbose]: /home/boobs/osu/osu.Desktop/bin/Debug/net8.0/runtimes/linux-x64/native/liblibdl.so: cannot open shared object file: No such file or directory
[runtime] 2024-04-16 19:35:14 [verbose]: /usr/share/dotnet/shared/Microsoft.NETCore.App/8.0.4/liblibdl.so: cannot open shared object file: No such file or directory
[runtime] 2024-04-16 19:35:14 [verbose]: /home/boobs/osu/osu.Desktop/bin/Debug/net8.0/liblibdl.so: cannot open shared object file: No such file or directory
[runtime] 2024-04-16 19:35:14 [verbose]: /home/boobs/osu/osu.Desktop/bin/Debug/net8.0/runtimes/linux-x64/native/libdl: cannot open shared object file: No such file or directory
[runtime] 2024-04-16 19:35:14 [verbose]: /usr/share/dotnet/shared/Microsoft.NETCore.App/8.0.4/libdl: cannot open shared object file: No such file or directory
[runtime] 2024-04-16 19:35:14 [verbose]: /home/boobs/osu/osu.Desktop/bin/Debug/net8.0/libdl: cannot open shared object file: No such file or directory
[runtime] 2024-04-16 19:35:14 [verbose]: /home/boobs/osu/osu.Desktop/bin/Debug/net8.0/runtimes/linux-x64/native/liblibdl: cannot open shared object file: No such file or directory
[runtime] 2024-04-16 19:35:14 [verbose]: /usr/share/dotnet/shared/Microsoft.NETCore.App/8.0.4/liblibdl: cannot open shared object file: No such file or directory
[runtime] 2024-04-16 19:35:14 [verbose]: /home/boobs/osu/osu.Desktop/bin/Debug/net8.0/liblibdl: cannot open shared object file: No such file or directory
[runtime] 2024-04-16 19:35:14 [verbose]: 
[runtime] 2024-04-16 19:35:14 [verbose]: at Vulkan.Libdl.dlerror()
[runtime] 2024-04-16 19:35:14 [verbose]: at Vulkan.NativeLibrary.UnixNativeLibrary.LoadLibrary(String libraryName)
[runtime] 2024-04-16 19:35:14 [verbose]: at Vulkan.NativeLibrary..ctor(String libraryName)
[runtime] 2024-04-16 19:35:14 [verbose]: at Vulkan.NativeLibrary.UnixNativeLibrary..ctor(String libraryName)
[runtime] 2024-04-16 19:35:14 [verbose]: at Vulkan.NativeLibrary.Load(String libraryName)
[runtime] 2024-04-16 19:35:14 [verbose]: at Vulkan.VulkanNative.LoadNativeLibrary()
[runtime] 2024-04-16 19:35:14 [verbose]: at Vulkan.VulkanNative..cctor()
[runtime] 2024-04-16 19:35:14 [verbose]: --- End of inner exception stack trace ---
[runtime] 2024-04-16 19:35:14 [verbose]: at Vulkan.VulkanNative.vkEnumerateInstanceLayerProperties(UInt32& pPropertyCount, VkLayerProperties* pProperties)
[runtime] 2024-04-16 19:35:14 [verbose]: at Veldrid.Vk.VulkanUtil.EnumerateInstanceLayers()
[runtime] 2024-04-16 19:35:14 [verbose]: at Veldrid.Vk.VkGraphicsDevice.CreateInstance(Boolean debug, VulkanDeviceOptions options)
[runtime] 2024-04-16 19:35:14 [verbose]: at Veldrid.Vk.VkGraphicsDevice..ctor(GraphicsDeviceOptions options, Nullable`1 scDesc, VulkanDeviceOptions vkOptions)
[runtime] 2024-04-16 19:35:14 [verbose]: at Veldrid.Vk.VkGraphicsDevice..ctor(GraphicsDeviceOptions options, Nullable`1 scDesc)
[runtime] 2024-04-16 19:35:14 [verbose]: at Veldrid.GraphicsDevice.CreateVulkan(GraphicsDeviceOptions options, SwapchainDescription swapchainDescription)
[runtime] 2024-04-16 19:35:14 [verbose]: at osu.Framework.Graphics.Veldrid.VeldridDevice..ctor(IGraphicsSurface graphicsSurface)
[runtime] 2024-04-16 19:35:14 [verbose]: at osu.Framework.Graphics.Rendering.Deferred.DeferredRenderer.Initialise(IGraphicsSurface graphicsSurface)
[runtime] 2024-04-16 19:35:14 [verbose]: at osu.Framework.Graphics.Rendering.Renderer.osu.Framework.Graphics.Rendering.IRenderer.Initialise(IGraphicsSurface graphicsSurface)
[runtime] 2024-04-16 19:35:14 [verbose]: at osu.Framework.Platform.GameHost.SetupRendererAndWindow(IRenderer renderer, GraphicsSurfaceType surfaceType)
[runtime] 2024-04-16 19:35:14 [important]: The selected renderer (Vulkan (Experimental)) failed to initialise. Renderer selection has been reverted to automatic.
[runtime] 2024-04-16 19:35:14 [verbose]: 🖼 Initialising "GL" renderer with "OpenGL" surface
[runtime] 2024-04-16 19:35:14 [verbose]: Updated display mode to desktop resolution: 2560x1440@165, SDL_PIXELFORMAT_RGB888
[runtime] 2024-04-16 19:35:14 [verbose]: GL Initialized
[runtime] 2024-04-16 19:35:14 [verbose]: GL Version:                 4.6 (Core Profile) Mesa 24.0.5-arch1.1
[runtime] 2024-04-16 19:35:14 [verbose]: GL Renderer:                AMD Radeon Graphics (radeonsi, raphael_mendocino, LLVM 17.0.6, DRM 3.57, 6.8.5-arch1-1)
[runtime] 2024-04-16 19:35:14 [verbose]: GL Shader Language version: 4.60
[runtime] 2024-04-16 19:35:14 [verbose]: GL Vendor:                  AMD
[runtime] 2024-04-16 19:35:14 [verbose]: GL Extensions:              GL_AMD_conservative_depth GL_AMD_depth_clamp_separate GL_AMD_draw_buffers_blend GL_AMD_framebuffer_multisample_advanced GL_AMD_gpu_shader_int64 GL_AMD_multi_draw_indirect GL_AMD_performance_monitor GL_AMD_pinned_memory GL_AMD_query_buffer_object GL_AMD_seamless_cubemap_per_texture GL_AMD_shader_stencil_export GL_AMD_shader_trinary_minmax GL_AMD_texture_texture4 GL_AMD_vertex_shader_layer GL_AMD_vertex_shader_viewport_index GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_ARB_ES2_compatibility GL_ARB_ES3_1_compatibility GL_ARB_ES3_2_compatibility GL_ARB_ES3_compatibility GL_ARB_arrays_of_arrays GL_ARB_base_instance GL_ARB_bindless_texture GL_ARB_blend_func_extended GL_ARB_buffer_storage GL_ARB_clear_buffer_object GL_ARB_clear_texture GL_ARB_clip_control GL_ARB_color_buffer_float GL_ARB_compressed_texture_pixel_storage GL_ARB_compute_shader GL_ARB_compute_variable_group_size GL_ARB_conditional_render_inverted GL_ARB_conservative_depth GL_ARB_copy_buffer GL_ARB_copy_image GL_ARB_cull_distance GL_ARB_debug_output GL_ARB_depth_buffer_float GL_ARB_depth_clamp GL_ARB_derivative_control GL_ARB_direct_state_access GL_ARB_draw_buffers GL_ARB_draw_buffers_blend GL_ARB_draw_elements_base_vertex GL_ARB_draw_indirect GL_ARB_draw_instanced GL_ARB_enhanced_layouts GL_ARB_explicit_attrib_location GL_ARB_explicit_uniform_location GL_ARB_fragment_coord_conventions GL_ARB_fragment_layer_viewport GL_ARB_fragment_shader GL_ARB_framebuffer_no_attachments GL_ARB_framebuffer_object GL_ARB_framebuffer_sRGB GL_ARB_get_program_binary GL_ARB_get_texture_sub_image GL_ARB_gl_spirv GL_ARB_gpu_shader5 GL_ARB_gpu_shader_fp64 GL_ARB_gpu_shader_int64 GL_ARB_half_float_pixel GL_ARB_half_float_vertex GL_ARB_indirect_parameters GL_ARB_instanced_arrays GL_ARB_internalformat_query GL_ARB_internalformat_query2 GL_ARB_invalidate_subdata GL_ARB_map_buffer_alignment GL_ARB_map_buffer_range GL_ARB_multi_bind GL_ARB_multi_draw_indirect GL_ARB_occlusion_query2 GL_ARB_parallel_shader_compile GL_ARB_pipeline_statistics_query GL_ARB_pixel_buffer_object GL_ARB_point_sprite GL_ARB_polygon_offset_clamp GL_ARB_post_depth_coverage GL_ARB_program_interface_query GL_ARB_provoking_vertex GL_ARB_query_buffer_object GL_ARB_robust_buffer_access_behavior GL_ARB_robustness GL_ARB_sample_shading GL_ARB_sampler_objects GL_ARB_seamless_cube_map GL_ARB_seamless_cubemap_per_texture GL_ARB_separate_shader_objects GL_ARB_shader_atomic_counter_ops GL_ARB_shader_atomic_counters GL_ARB_shader_ballot GL_ARB_shader_bit_encoding GL_ARB_shader_clock GL_ARB_shader_draw_parameters GL_ARB_shader_group_vote GL_ARB_shader_image_load_store GL_ARB_shader_image_size GL_ARB_shader_objects GL_ARB_shader_precision GL_ARB_shader_stencil_export GL_ARB_shader_storage_buffer_object GL_ARB_shader_subroutine GL_ARB_shader_texture_image_samples GL_ARB_shader_texture_lod GL_ARB_shader_viewport_layer_array GL_ARB_shading_language_420pack GL_ARB_shading_language_include GL_ARB_shading_language_packing GL_ARB_sparse_buffer GL_ARB_sparse_texture GL_ARB_sparse_texture2 GL_ARB_sparse_texture_clamp GL_ARB_spirv_extensions GL_ARB_stencil_texturing GL_ARB_sync GL_ARB_tessellation_shader GL_ARB_texture_barrier GL_ARB_texture_border_clamp GL_ARB_texture_buffer_object GL_ARB_texture_buffer_object_rgb32 GL_ARB_texture_buffer_range GL_ARB_texture_compression_bptc GL_ARB_texture_compression_rgtc GL_ARB_texture_cube_map_array GL_ARB_texture_filter_anisotropic GL_ARB_texture_float GL_ARB_texture_gather GL_ARB_texture_mirror_clamp_to_edge GL_ARB_texture_multisample GL_ARB_texture_non_power_of_two GL_ARB_texture_query_levels GL_ARB_texture_query_lod GL_ARB_texture_rectangle GL_ARB_texture_rg GL_ARB_texture_rgb10_a2ui GL_ARB_texture_stencil8 GL_ARB_texture_storage GL_ARB_texture_storage_multisample GL_ARB_texture_swizzle GL_ARB_texture_view GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ARB_transform_feedback_instanced GL_ARB_transform_feedback_overflow_query GL_ARB_uniform_buffer_object GL_ARB_vertex_array_bgra GL_ARB_vertex_array_object GL_ARB_vertex_attrib_64bit GL_ARB_vertex_attrib_binding GL_ARB_vertex_buffer_object GL_ARB_vertex_shader GL_ARB_vertex_type_10f_11f_11f_rev GL_ARB_vertex_type_2_10_10_10_rev GL_ARB_viewport_array GL_ATI_blend_equation_separate GL_ATI_meminfo GL_ATI_texture_float GL_ATI_texture_mirror_once GL_EXT_EGL_image_storage GL_EXT_EGL_sync GL_EXT_abgr GL_EXT_blend_equation_separate GL_EXT_debug_label GL_EXT_demote_to_helper_invocation GL_EXT_depth_bounds_test GL_EXT_draw_buffers2 GL_EXT_draw_instanced GL_EXT_framebuffer_blit GL_EXT_framebuffer_multisample GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_framebuffer_object GL_EXT_framebuffer_sRGB GL_EXT_memory_object GL_EXT_memory_object_fd GL_EXT_packed_depth_stencil GL_EXT_packed_float GL_EXT_pixel_buffer_object GL_EXT_polygon_offset_clamp GL_EXT_provoking_vertex GL_EXT_semaphore GL_EXT_semaphore_fd GL_EXT_shader_image_load_formatted GL_EXT_shader_image_load_store GL_EXT_shader_integer_mix GL_EXT_shader_samples_identical GL_EXT_texture_array GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_compression_s3tc GL_EXT_texture_filter_anisotropic GL_EXT_texture_integer GL_EXT_texture_mirror_clamp GL_EXT_texture_sRGB GL_EXT_texture_sRGB_R8 GL_EXT_texture_sRGB_RG8 GL_EXT_texture_sRGB_decode GL_EXT_texture_shadow_lod GL_EXT_texture_shared_exponent GL_EXT_texture_snorm GL_EXT_texture_swizzle GL_EXT_timer_query GL_EXT_transform_feedback GL_EXT_vertex_array_bgra GL_EXT_vertex_attrib_64bit GL_EXT_window_rectangles GL_IBM_multimode_draw_arrays GL_INTEL_blackhole_render GL_KHR_blend_equation_advanced GL_KHR_context_flush_control GL_KHR_debug GL_KHR_no_error GL_KHR_parallel_shader_compile GL_KHR_robust_buffer_access_behavior GL_KHR_robustness GL_KHR_texture_compression_astc_ldr GL_KHR_texture_compression_astc_sliced_3d GL_MESA_framebuffer_flip_y GL_MESA_pack_invert GL_MESA_shader_integer_functions GL_MESA_texture_const_bandwidth GL_MESA_texture_signed_rgba GL_NVX_gpu_memory_info GL_NV_alpha_to_coverage_dither_control GL_NV_compute_shader_derivatives GL_NV_conditional_render GL_NV_copy_image GL_NV_depth_clamp GL_NV_packed_depth_stencil GL_NV_shader_atomic_int64 GL_NV_texture_barrier GL_NV_vdpau_interop GL_OES_EGL_image GL_S3_s3tc
[runtime] 2024-04-16 19:35:14 [verbose]: UseStructuredBuffers: True
[runtime] 2024-04-16 19:35:14 [verbose]: 🖼 Renderer initialised!
[runtime] 2024-04-16 19:35:14 [verbose]: Host execution state changed to Running
[runtime] 2024-04-16 19:35:14 [verbose]: Execution mode changed to MultiThreaded
[input] 2024-04-16 19:35:14 [verbose]: [MIDI] Connected MIDI device: Midi Through Port-0
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablets...
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet '10moon 1060N'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Acepen AP 1060'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Adesso Cybertablet K8'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Artisul A1201'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Artisul AP604'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Artisul M0610 Pro'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'FlooGoo FMA100'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon 1060 Pro'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon GM116HD'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon GM156HD'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon M106K Pro'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon M106K'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon M10K Pro'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon M10K'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon M1220'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon M1230'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon M6'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon PD1161'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon PD156 Pro'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon PD1560'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon PD1561'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon PD2200'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon S56K'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon S620'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon S630'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Gaomon S830'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Genius G-Pen 560'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Genius i405x'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Genius i608x'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion 420'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion G10T'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion G930L'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion GC610'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion GT-220 V2'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion GT-221 Pro'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion GT-221'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion H1060P'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion H1161'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion H320M'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion H420'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion H420X'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion H430P'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion H580X'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion H610 Pro V2'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion H610 Pro V3'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion H610 Pro'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion H610X'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion H640P'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion H641P'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion H642'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion H690'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion H950P'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion HC16'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion HS610'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion HS611'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion HS64'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion HS95'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Kamvas 12'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Kamvas 13'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Kamvas 16 (2021)'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Kamvas 16'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Kamvas 20'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Kamvas 22 Plus'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Kamvas 22'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Kamvas 24 Plus'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Kamvas Pro 12'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Kamvas Pro 13 (2.5k)'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Kamvas Pro 13'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Kamvas Pro 16 (2.5k)'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Kamvas Pro 16'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Kamvas Pro 20'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Kamvas Pro 24 (4K)'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Kamvas Pro 24'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion New 1060 Plus (2048)'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion New 1060 Plus'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Q11K V2'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Q11K'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Q620M'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion Q630M'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion RDS-160'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion RTE-100'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion RTM-500'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion WH1409 V2'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Huion WH1409'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'KENTING K5540'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'LifeTec LT9570'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Monoprice 10594'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Monoprice MP1060-HA60'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Parblo A609'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Parblo A610 Pro'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Parblo A640 V2'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Parblo Intangbo M'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Parblo Intangbo S'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Parblo Ninos M'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Parblo Ninos N4'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Parblo Ninos N7'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Parblo Ninos S'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'RobotPen T9A'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'UC-Logic 1060N'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'UC-Logic PF1209'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'UGEE S640'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'UGEE U1200'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'UGEE U1600'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'UGTABLET M708 V2'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'UGTABLET M708'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'VEIKK A15 Pro'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'VEIKK A15 V2'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'VEIKK A15'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'VEIKK A30 V2'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'VEIKK A30'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'VEIKK A50 V2'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'VEIKK A50'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'VEIKK S640 V2'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'VEIKK S640'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'VEIKK VK1060'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'VEIKK VK1060PRO'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'VEIKK VK430 V2'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'VEIKK VK430'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'VEIKK VK640'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'VEIKK Viola (VO1060)'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'ViewSonic Woodpad PF0730'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTC-4110WL'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTC-6110WL'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTE-430'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTE-440'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTE-450'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTE-460'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTE-630'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTE-640'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTE-650'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTE-660'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTF-430'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTH-301'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTH-460'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTH-461'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTH-470'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTH-480'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTH-490'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTH-661'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTH-670'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTH-680'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTH-690'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTL-4100'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTL-4100WL'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTL-460'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTL-470'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTL-471'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTL-472'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTL-480'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTL-490'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTL-6100'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTL-6100WL'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTL-671'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTL-672'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTL-680'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom CTL-690'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom DTC-133'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom DTH-1320'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom Cintiq 13HD (DTK-1300)'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom Cintiq 16 (DTK1660)'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom Cintiq 12WX (DTZ-1200W)'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom ET-0405-U'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom ET-0405A-U'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom FT-0405-U'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom GD-0405-U'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom GD-0608-U'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom GD-0912-U'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom GD-1212-U'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom GD-1218-U'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom MTE-450'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTH-450'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTH-451'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTH-460'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTH-650'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTH-651'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTH-660'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTH-850'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTH-851'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTH-860'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTK-1240'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTK-440'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTK-450'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTK-540WL'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTK-640'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTK-650'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTK-840'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTU-600U'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTZ-1230'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTZ-1231W'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTZ-430'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTZ-431W'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTZ-630'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTZ-631W'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom PTZ-930'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom XD-0405-U'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom XD-0608-U'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom XD-0912-U'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom XD-1212-U'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Wacom XD-1218-U'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'Waltop Slim Tablet 5.8"'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XenceLabs Pen Tablet Medium'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XenceLabs Pen Tablet Small'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XENX P1-640'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XENX P3-1060'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XENX X1-640'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Artist 10 (2nd Gen)'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Artist 12 (2nd Gen)'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Artist 12 Pro'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Artist 12'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Artist 13 (2nd Gen)'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Artist 13.3 Pro'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Artist 13.3'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Artist 15.6 Pro'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Artist 15.6'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Artist 16 (2nd Gen)'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Artist 16 Pro'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Artist 22HD'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Artist Pro 16TP'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen CT1060'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen CT430'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen CT640'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Deco 01 V2 (variant 2)'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Deco 01 V2'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Deco 01'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Deco 02'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Deco 03'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Deco L'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Deco M'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Deco mini4'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Deco mini7'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Deco Pro Medium'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Deco Pro Small'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Deco Pro SW'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Innovator 16'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Star 03'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Star 05 V3'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Star 06'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Star 06C'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Star G430'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Star G430S V2'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Star G430S'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Star G540 Pro'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Star G540'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Star G640 V2'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Star G640'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Star G640S'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Star G960'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Star G960S Plus'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: Searching for tablet 'XP-Pen Star G960S'
[input] 2024-04-16 19:35:14 [verbose]: [Tablet] Detect: No tablets were detected.
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font FontAwesome-Solid...
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Roboto-Regular...
[runtime] 2024-04-16 19:35:14 [verbose]: 🔈 BASS initialised
[runtime] 2024-04-16 19:35:14 [verbose]: BASS version:           2.4.16.7
[runtime] 2024-04-16 19:35:14 [verbose]: BASS FX version:        2.4.12.6
[runtime] 2024-04-16 19:35:14 [verbose]: BASS MIX version:       2.4.11.1
[runtime] 2024-04-16 19:35:14 [verbose]: Device:                 Default
[runtime] 2024-04-16 19:35:14 [verbose]: Driver:                 default
[runtime] 2024-04-16 19:35:14 [verbose]: Update period:          5 ms
[runtime] 2024-04-16 19:35:14 [verbose]: Device buffer length:   10 ms
[runtime] 2024-04-16 19:35:14 [verbose]: Playback buffer length: 100 ms
[runtime] 2024-04-16 19:35:14 [verbose]: 🔈 BASS initialised
[runtime] 2024-04-16 19:35:14 [verbose]: BASS version:           2.4.16.7
[runtime] 2024-04-16 19:35:14 [verbose]: BASS FX version:        2.4.12.6
[runtime] 2024-04-16 19:35:14 [verbose]: BASS MIX version:       2.4.11.1
[runtime] 2024-04-16 19:35:14 [verbose]: Device:                 Default
[runtime] 2024-04-16 19:35:14 [verbose]: Driver:                 default
[runtime] 2024-04-16 19:35:14 [verbose]: Update period:          5 ms
[runtime] 2024-04-16 19:35:14 [verbose]: Device buffer length:   10 ms
[runtime] 2024-04-16 19:35:14 [verbose]: Playback buffer length: 100 ms
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font FontAwesome-Solid!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font FontAwesome-Regular...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font FontAwesome-Regular!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font FontAwesome-Brands...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font FontAwesome-Brands!
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font Roboto-Regular!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Roboto-RegularItalic...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font Roboto-RegularItalic!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Roboto-Bold...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font Roboto-Bold!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Roboto-BoldItalic...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font Roboto-BoldItalic!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font RobotoCondensed-Regular...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font RobotoCondensed-Regular!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font RobotoCondensed-Bold...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font RobotoCondensed-Bold!
[runtime] 2024-04-16 19:35:14 [verbose]: Beginning realm file store cleanup
[runtime] 2024-04-16 19:35:14 [verbose]: Finished realm file store cleanup (0 of 0 deleted)
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font osuFont...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font osuFont!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Torus-Regular...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font Torus-Regular!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Torus-Light...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font Torus-Light!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Torus-SemiBold...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font Torus-SemiBold!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Torus-Bold...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font Torus-Bold!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Torus-Alternate-Regular...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font Torus-Alternate-Regular!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Torus-Alternate-Light...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font Torus-Alternate-Light!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Torus-Alternate-SemiBold...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font Torus-Alternate-SemiBold!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Torus-Alternate-Bold...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font Torus-Alternate-Bold!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Inter-Regular...
[runtime] 2024-04-16 19:35:14 [verbose]: Opened realm "/home/boobs/.local/share/osu-development/client_40.realm" at version 40
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font Inter-Regular!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Inter-RegularItalic...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font Inter-RegularItalic!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Inter-Light...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font Inter-Light!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Inter-LightItalic...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font Inter-LightItalic!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Inter-SemiBold...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font Inter-SemiBold!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Inter-SemiBoldItalic...
[runtime] 2024-04-16 19:35:14 [debug]: Loaded Font Inter-SemiBoldItalic!
[runtime] 2024-04-16 19:35:14 [debug]: Loading Font Inter-Bold...
Console output is being limited. Please check 1713296114.runtime.log for full logs.
[performance] 2024-04-16 19:35:14 [verbose]: TextureAtlas initialised (1024x1024)
[runtime] 2024-04-16 19:35:14 [verbose]: Updated display mode to fullscreen resolution: 2560x1440@165, SDL_PIXELFORMAT_RGB888
[performance] 2024-04-16 19:35:14 [verbose]: TextureAtlas initialised (1024x1024)
[runtime] 2024-04-16 19:35:15 [verbose]: Loading FPSCounter...
/usr/bin/xdg-mime: line 885: qtpaths: command not found
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded FPSCounter!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading VersionManager...
[runtime] 2024-04-16 19:35:15 [verbose]: Texture 3's upload queue is large (100)
[performance] 2024-04-16 19:35:15 [verbose]: Texture requested (1920x8) which exceeds TextureStore's atlas size (1024x1024) - bypassing atlasing. Consider using LargeTextureStore.
[runtime] 2024-04-16 19:35:15 [verbose]: Texture 3's upload queue is large (100)
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded VersionManager!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading OsuLogo...
[runtime] 2024-04-16 19:35:15 [verbose]: Texture 3's upload queue is large (200)
[performance] 2024-04-16 19:35:15 [verbose]: Texture requested (1024x1024) which exceeds TextureStore's atlas size (1024x1024) - bypassing atlasing. Consider using LargeTextureStore.
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded OsuLogo!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading UserStatisticsWatcher...
[runtime] 2024-04-16 19:35:15 [verbose]: 📺 OsuScreenStack#253(depth:1) loading Loader#392
[runtime] 2024-04-16 19:35:15 [verbose]: 📺 OsuScreenStack#253(depth:1) entered Loader#392
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded UserStatisticsWatcher!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading Toolbar...
[runtime] 2024-04-16 19:35:15 [verbose]: Texture 3's upload queue is large (100)
[runtime] 2024-04-16 19:35:15 [verbose]: Texture 3's upload queue is large (100)
[runtime] 2024-04-16 19:35:15 [verbose]: 📺 OsuScreenStack#253(depth:1) suspended Loader#392 (waiting on IntroTriangles#398)
[runtime] 2024-04-16 19:35:15 [verbose]: 📺 OsuScreenStack#253(depth:2) entered IntroTriangles#398
[runtime] 2024-04-16 19:35:15 [verbose]: 📺 BackgroundScreenStack#298(depth:1) loading BackgroundScreenDefault#355
[runtime] 2024-04-16 19:35:15 [verbose]: Game-wide working beatmap updated to cYsmix - triangles (peppy) [peppy]
[runtime] 2024-04-16 19:35:15 [verbose]: 🌅 Global background change queued
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded Toolbar!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading VolumeOverlay...
[runtime] 2024-04-16 19:35:15 [verbose]: 📺 BackgroundScreenStack#298(depth:1) entered BackgroundScreenDefault#355
[runtime] 2024-04-16 19:35:15 [verbose]: Texture 3's upload queue is large (100)
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded VolumeOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading OnScreenDisplay...
[runtime] 2024-04-16 19:35:15 [verbose]: Texture 3's upload queue is large (200)
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded OnScreenDisplay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading NotificationOverlay...
[runtime] 2024-04-16 19:35:15 [verbose]: decoupled ruleset transferred ("" -> "osu!")
[performance] 2024-04-16 19:35:15 [verbose]: TextureAtlas size exceeded 1 time(s); generating new texture (1024x1024)
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded NotificationOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading LegacyImportManager...
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded LegacyImportManager!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading ScreenshotManager...
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded ScreenshotManager!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading SimpleUpdateManager...
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded SimpleUpdateManager!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading FirstRunSetupOverlay...
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded FirstRunSetupOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading ManageCollectionsDialog...
[runtime] 2024-04-16 19:35:15 [verbose]: Texture 8's upload queue is large (100)
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded ManageCollectionsDialog!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading BeatmapListingOverlay...
[runtime] 2024-04-16 19:35:15 [verbose]: Texture 8's upload queue is large (100)
[runtime] 2024-04-16 19:35:15 [verbose]: Texture 8's upload queue is large (100)
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded BeatmapListingOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading DashboardOverlay...
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded DashboardOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading NewsOverlay...
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded NewsOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading RankingsOverlay...
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded RankingsOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading ChannelManager...
[network] 2024-04-16 19:35:15 [verbose]: Failing request osu.Game.Online.API.Requests.GetUpdatesRequest (System.Net.WebException: User not logged in)
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded ChannelManager!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading ChatOverlay...
[performance] 2024-04-16 19:35:15 [verbose]: TextureAtlas size exceeded 2 time(s); generating new texture (1024x1024)
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded ChatOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading MessageNotifier...
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded MessageNotifier!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading SettingsOverlay...
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded SettingsOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading ChangelogOverlay...
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded ChangelogOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading UserProfileOverlay...
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded UserProfileOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading BeatmapSetOverlay...
[runtime] 2024-04-16 19:35:15 [verbose]: 🌅 Global background loading
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded BeatmapSetOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading WikiOverlay...
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded WikiOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading SkinEditorOverlay...
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded SkinEditorOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading LoginOverlay...
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded LoginOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading NowPlayingOverlay...
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded NowPlayingOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading AccountCreationOverlay...
[runtime] 2024-04-16 19:35:15 [verbose]: 📺 ScreenStack#774(depth:1) scheduling push ScreenWelcome#882
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded AccountCreationOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading DialogOverlay...
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded DialogOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading MedalOverlay...
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded MedalOverlay!
[runtime] 2024-04-16 19:35:15 [verbose]: Loading BackgroundDataStoreProcessor...
[runtime] 2024-04-16 19:35:15 [verbose]: Beginning background data store processing..
[runtime] 2024-04-16 19:35:15 [verbose]: Querying for beatmap sets to reprocess...
[runtime] 2024-04-16 19:35:15 [verbose]: Querying for beatmaps with missing hitobject counts to reprocess...
[runtime] 2024-04-16 19:35:15 [verbose]: Querying for scores to reprocess...
[runtime] 2024-04-16 19:35:15 [verbose]: Querying for scores that need total score conversion...
[runtime] 2024-04-16 19:35:15 [verbose]: Found 0 scores which require total score conversion.
[runtime] 2024-04-16 19:35:15 [verbose]: Querying for scores that need rank upgrades...
[runtime] 2024-04-16 19:35:15 [verbose]: Found 0 scores which require rank upgrades.
[runtime] 2024-04-16 19:35:15 [verbose]: Finished background data store processing!
[runtime] 2024-04-16 19:35:15 [verbose]: Loaded BackgroundDataStoreProcessor!
[runtime] 2024-04-16 19:35:18 [verbose]: 📺 OsuScreenStack#253(depth:2) suspended IntroTriangles#398 (waiting on MainMenu#125)
[runtime] 2024-04-16 19:35:18 [verbose]: 📺 OsuScreenStack#253(depth:3) entered MainMenu#125
[network] 2024-04-16 19:35:18 [verbose]: Request to https://assets.ppy.sh/menu-content.json successfully completed!
[runtime] 2024-04-16 19:35:18 [verbose]: ⚠ The selected renderer (Vulkan (Experimental)) failed to initialise. Renderer selection has been reverted to automatic.
[network] 2024-04-16 19:35:18 [verbose]: Request to https://assets.ppy.sh/main-menu/project-loved-2@2x.png successfully completed!
[runtime] 2024-04-16 19:35:18 [verbose]: ButtonSystem's state changed from Initial to TopLevel
[runtime] 2024-04-16 19:35:26 [verbose]: ButtonSystem's state changed from TopLevel to Initial


```

### Version

latest dev 16.04.24 and  2024.412.1

### Logs

[1713296386.input.log](https://github.com/ppy/osu/files/15001860/1713296386.input.log)
[1713296386.network.log]
[1713296413.network.log](https://github.com/ppy/osu/files/15001865/1713296413.network.log)
[1713296413.performance.log](https://github.com/ppy/osu/files/15001866/1713296413.performance.log)
[1713296413.runtime.log](https://github.com/ppy/osu/files/15001867/1713296413.runtime.log)
(https://github.com/ppy/osu/files/15001861/1713296386.network.log)
[Uploading 1713296386.perform
[1713296386.runtime.log](https://github.com/ppy/osu/files/15001863/1713296386.runtime.log)
[1713296413.input.log](https://github.com/ppy/osu/files/15001864/1713296413.input.log)
ance.log…]()

**Comment by 64ArthurAraujo** [comment]
Did you try https://github.com/ppy/osu/discussions/27659#discussioncomment-9101487? it seems that this is the same problem.
